lambda/lexv2-build/handler.py

```diff
@@ -28,17 +28,78 @@
 LEXV2_BOT_DRAFT_VERSION = "DRAFT"
 LEXV2_TEST_BOT_ALIAS = "TestBotAlias"
 LEXV2_BOT_LOCALE_VOICES = {
-	"de_DE": "Hans",
-	"en_AU": "Nicole",
-	"en_GB": "Amy",
-	"en_US": "Joanna",
-	"es_419": "Mia",
-	"es_ES": "Conchita",
-	"es_US": "Lupe",
-	"fr_CA": "Chantal",
-	"fr_FR": "Mathieu",
-	"it_IT": "Bianca",
-	"ja_JP": "Mizuki"
+	"de_AT": [{
```

Comment:
Love that we can now use neural voices! Curious why you made this an array of dicts rather than just a dict per locale? Do you anticipate supporting multiple voices per locale later? Not sure how that would work.  It's fine - just curious why..

Reply:
Initially, was thinking if we could include a parameter in CloudFormation for users to select the engine type (standard | neural) as a option in the setup. So thought the array of dicts could potentially hold multiple engine types if needed for a given language locale.

Reply:
Makes sense - smart thinking!